File: snippets/python/set_payload.py
from qdrant_client import QdrantClient

client = QdrantClient(url="http://localhost:6333")

client.set_payload(
    collection_name="{collection_name}",
    payload={
        "property1": "string",
        "property2": "string",
    },
    points=[0, 3, 10],
)

File: snippets/typescript/set_payload.ts
import { QdrantClient } from "@qdrant/js-client-rest";

const client = new QdrantClient({ host: "localhost", port: 6333 });

client.setPayload("global_patient_data", {
  payload: {
    is_smoker: false,
    gender: "Male",
  },
  points: [0, 3, 10],
});
